model/host/host_test.go

```diff
@@ -801,6 +801,35 @@ func TestFindByLCT(t *testing.T) {
 	})
 }
 
+func TestHostElapsedCommTime(t *testing.T) {
+	assert := assert.New(t)
+	now := time.Now()
+	hostThatRanTask := Host{
+		Id:                    "hostThatRanTask",
+		CreationTime:          now.Add(-30 * time.Minute),
+		StartTime:             now.Add(-20 * time.Minute),
+		LastCommunicationTime: now.Add(-10 * time.Minute),
+	}
+	hostThatJustStarted := Host{
+		Id:           "hostThatJustStarted",
+		CreationTime: now.Add(-5 * time.Minute),
+		StartTime:    now.Add(-1 * time.Minute),
+	}
+	hostWithNoCreateTime := Host{
+		Id: "hostWithNoCreateTime",
+		LastCommunicationTime: now.Add(-15 * time.Minute),
+	}
+	hostWithOnlyCreateTime := Host{
+		Id:           "hostWithOnlyCreateTime",
+		CreationTime: now.Add(-7 * time.Minute),
+	}
+
+	assert.InDelta(float64(10*time.Minute), float64(hostThatRanTask.GetElapsedCommunicationTime()), float64(1*time.Millisecond))
```

Comment:
Aren't these values all `time.Duration`s? Why convert? Also, since you're saving `now`, is `InDelta` necessary?

Reply:
Trying to compare durations converts to the string representation (ie. 10m15s) so I need to convert. The GetElapsedCommunicationTime function uses its own time.Now() so I need to fuzz the difference